packages/ui/src/elements/WhereBuilder/Condition/Relationship/index.tsx

```diff
@@ -209,7 +205,9 @@ export const RelationshipField: React.FC<Props> = (props) => {
   }, [hasMany, hasMultipleRelations, value, options])
 
   const handleInputChange = (input: string) => {
+    dispatchOptions({ type: 'CLEAR', i18n, required: false })
```

Comment:
If this line is not added, when writing something the results that are in the select will not disappear, even if they do not meet the new search term.